internal/pkg/skuba/upgrade/addon/version.go

```diff
@@ -46,7 +46,8 @@ func UpdatedAddons(clusterVersion *version.Version) (AddonVersionInfoUpdate, err
 	for addonName, version := range latestAddonVersions {
 		skubaConfigVersion := skubaConfig.AddonsVersion[addonName]
 		aviu.Current[addonName] = skubaConfigVersion
-		if version.Version > skubaConfigVersion.Version || version.ManifestVersion > skubaConfigVersion.ManifestVersion {
+		if skubaConfigVersion == nil ||
+			skubaConfigVersion != nil && (version.Version > skubaConfigVersion.Version || version.ManifestVersion > skubaConfigVersion.ManifestVersion) {
```

Comment:
As per `(version.Version > skubaConfigVersion.Version || version.ManifestVersion > skubaConfigVersion.ManifestVersion)`, it seems we don't need to update manifest version if only image version updated.

Ref: @ereslibre https://github.com/SUSE/skuba/pull/726#pullrequestreview-293964654

Reply:
It's more tied to the RFC reasoning than to the actual implementation. You are right, but I prefer if we always bump the manifest version as well (we can trace every possible change to a univocally defined manifest version).

Reply:
Understood and make sense to me. Thanks!

Reply:
We bump into an issue with version `3.1.0-rev4` is **LessThan** version `3.1.0`, so I think only bump the manifest version could solve this problem.

Reply:
This should be resolved by semantic version PR later.